avocado/core/messages.py

```diff
@@ -104,7 +104,18 @@ class StartMessageHandler(BaseMessageHandler):
     example: {'status': 'started', 'time': 16444.819830573}
     """
 
-    def handle(self, message, task, job):
+    @staticmethod
```

Comment:
Hi @richtja I haven't double checked this, but I remember something about streaming outputs for long results on stdout being one reason for this being set at the startup message. Have you considered that?

Reply:
Hi @beraldoleal, sorry but I don't understand your question. This is still set at the startup message. This commit only fixes cases when the Task finish before the startup because of timeout.

Reply:
Ohh so please, ignore my comment.